apps/rush-lib/src/logic/buildCache/CacheEntryId.ts

```diff
@@ -84,6 +86,31 @@ export class CacheEntryId {
             }
           }
 
+          case PHASE_NAME_TOKEN_NAME: {
+            switch (tokenAttribute) {
+              case undefined: {
+                throw new Error(
+                  'Either the "normalize" or the "trimPrefix" attribute is required ' +
+                    `for the "${tokenName}" token.`
+                );
+              }
+
+              case 'normalize': {
+                // Replace colons with underscores.
+                return `\${${OPTIONS_ARGUMENT_NAME}.phaseName.replace(/:/g, '_')}`;
+              }
+
+              case 'trimPrefix': {
+                // Trim the "_phase:" prefix from the phase name.
+                return `\${${OPTIONS_ARGUMENT_NAME}.phaseName.replace(/^_phase:/, '')}`;
```

Comment:
Is this the same operation as `_normalizeNameForLogFilenameIdentifiers()`?

Reply:
No, this strips the "_phase:" prefix. That replaces the ":" with "_"